templates/publisher/account-snaps.html

```diff
@@ -124,7 +124,7 @@ <h3 class="p-heading--five">Join the community</h3>
           {% for snap in snaps|sort %}
             {% with %}
               {% set dispute_pending = snaps[snap].status == "DisputePending" %}
-              {% set published = snaps[snap].status == "Published" %}
+              {% set published = snaps[snap].latest_revisions[0].status == "Published" %}
```

Comment:
It used to be like that before, was incorrectly changed to `snaps[snap].status` in https://github.com/canonical-websites/snapcraft.io/commit/b0d0144c6cf54e70e1955948e533f4af63a344e5#diff-a318410b3a4f551162a893952b3c01bb